release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -81,6 +81,13 @@ For more information, see xref:../installing/install_config/installing-customizi
 
 If a cloud administrator has already set a custom `/etc/chrony.conf` configuration, {op-system} no longer sets the `PEERNTP=no` option by default on cloud platforms. Otherwise, the `PEERNTP=no` option is still set by default. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1924869[BZ#1924869] for more information.
 
+[id="ocp-4-8-rhcos-"multipath-install"]
+==== Enabling multipathing at bare metal installation time
+
+Enabling multipathing during bare metal installation is now supported for nodes provisioned in {product-title} 4.8 or higher. You can enable multipathing by appending kernel arguments to the `coreos-installer install` command so that the installed system itself will use multipath beginning from the first boot. While post-installation support is still available by activating multipathing via the machine config, enabling multipathing during installation is recommended for nodes provisioned starting in {product-title} 4.8.
```

Comment:
```suggestion
Enabling multipathing during bare metal installation is now supported for nodes provisioned in {product-title} 4.8 or higher. You can enable multipathing by appending kernel arguments to the `coreos-installer install` command so that the installed system itself uses multipath beginning from the first boot. While post-installation support is still available by activating multipathing through the machine config, enabling multipathing during installation is recommended for nodes provisioned starting in {product-title} 4.8.
```

> ! via preposition
Use this term, which means "by way of", to refer to routing data, such as in "Messages are routed via SMTP". In other situations, try to replace "via" with one of the following synonyms: "across", "along", "by", "from", "on", or "through".

Reply:
Thanks, @kalexand-rh.

- I contemplated "uses" rather than "will use" in this particular case because it seemed a bit of a sequential process: append kargs now so that it will be used at first boot. I was probably overthinking it, and "uses" has the benefit of sounding fine and also conforming to ISG. Changed.
- I also contemplated replacing "via" but nothing sounded more appropriate in this case. IMO, "activating multipathing _through_ the machine config" makes it sound like it's actually traveling through the MC, which seems odd. In this occurrence, I take it to actually mean "by way of", so it seems "via" is the most appropriate. Unchanged.